src/Controllers/CodeController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using EWS.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace EWS;
+
+[ApiController]
+[Route("[controller]")]
+public class CodeController : EwsControllerBase<Code>
+{
+    public CodeController(EwsContext context)
+    : base(context)
+    {
+    }
+
+    [HttpGet]
+    public async Task<IEnumerable<Code>> GetByCodeTypAsync(int codeTypId)
```

Comment:
REQ: Codetyp konsequenterweise immer gleich schreiben:
```suggestion
    public async Task<IEnumerable<Code>> GetByCodetypAsync(int codetypId)
```